packages/angular-workspace/package.json

```diff
@@ -44,15 +44,15 @@
     "@angular/cli": "^16.2.12",
```

Comment:
@msmithNI think you could take a peak to see if there is an easy fix to push to the branch for the tanstack virtualizer usage that breaks the build in the PR?

Reply:
@rajsite pushed an update to fix the TanStack-virtualizer-related build errors.

Reply:
Much appreciated!